### PR TITLE
feat: basic functional prototype

### PR DESCRIPTION
**Pull request by agus-xyz**
## Changes

  - Add basic functional sugarrush prototype
  
### Checks

* [x] Have you catchup your branch with the latest state in base?
* [x] Have you lint your code locally prior to submission?
* [x] Have you reviewed your proposed changes and removed debris?
* [x] Have you successfully ran due tests with your changes?

### CODE REVIEW

File: components/Carousel.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import { FC, useState } from 'react'
```
Comment (evavirseda) [inline components/Carousel.tsx:1]: Carousel is an unnecessary component too

File: hooks/useMintCandyMachine.tsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+                    tokenMint: state.tokenMint,
+                    gatekeeper: state.data.gatekeeper,
+                    endSettings: state.data.endSettings,
+                    whitelistMintSettings: state.data.whitelistMintSettings,
+                    hiddenSettings: state.data.hiddenSettings,
+                    price: state.data.price,
+                    retainAuthority: state.data.retainAuthority,
+                },
+            })
+        } catch (_) {}
```
Comment (evavirseda) [inline hooks/useMintCandyMachine.tsx:73]: Wouldn't it be better if this `catch` contained something?

File: lib/interfaces.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export interface ICarousel {
```
Comment (evavirseda) [inline lib/interfaces.ts:1]: unnecessary

File: styles/modules/Button.module.scss [hunk 4]
```diff
@@ -0,0 +1,75 @@
+.modal {
```
Comment (evavirseda) [inline styles/modules/Button.module.scss:1]: Modal and button modules are unused

**Review by evavirseda — DISMISSED** [review]

**Comment by evavirseda** [comment]
This error appears when you create or update CM
`Error deploying config to Solana network. TypeError: Cannot assign to read only property '_nextClientSubscriptionId' of object '#<Connection>'`

**Comment by evavirseda** [comment]
> @evavirseda could you please address your feedback and fix it in a new branch? We will proceed to merge those fixes into `dev` before merging in `main`

sure!

**Comment by agus-xyz** [comment]
so, @evavirseda there is still an error when creating or updating a CM?

**Comment by evavirseda** [comment]
> so, @evavirseda there is still an error when creating or updating a CM?

yes, @cpl121 is working on it

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**